gateway/1.6/modules/ROOT/pages/flex-troubleshoot-soap-apis.adoc

```diff
@@ -0,0 +1,29 @@
+= Publishing SOAP APIs
```

Comment:
I'm thinking the title should match the other other troubleshooting pages, for consistency-sake.

```suggestion
= Troubleshooting SOAP APIs
```